src/App.js

```diff
@@ -1,24 +1,9 @@
-import logo from './logo.svg';
-import './App.css';
+import React from 'react';
+import Calculator from './components/Calculator';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <Calculator />
   );
 }
```

Comment:
- [ ] Please change this function to an arrow function to follow project requirements.
![image](https://user-images.githubusercontent.com/83803414/184089148-8a3ab3ec-dcb2-4f40-b586-e5d9ace90674.png)